partner_firstname/tests/__init__.py

```diff
@@ -35,5 +35,6 @@
     test_defaults,
     test_empty,
     test_name,
-    test_onchange
+    test_onchange,
+    test_order
```

Comment:
Asked by my obsessive–compulsive disorder, I have to tell you that if you add a comma, we reduce future diffs :grin: